### ISSUE TITLE
Error when not using webpacker: uninitialized constant SSLocal::Railtie::Webpacker

### ISSUE THREAD

**Issue by jaredmoody** [pat/sslocal-rb#1]
IDK if you're still maintaining this gem, but I'm still using it :D 

I'm working on a spike for transitioning from webpacker to jsbundling-rails, and running into an issue where sslocal tries to load webpacker when not installed:

```
NameError: uninitialized constant SSLocal::Railtie::Webpacker
/Users/jaredmoody/.asdf/installs/ruby/3.0.5/lib/ruby/gems/3.0.0/gems/sslocal-0.1.0/lib/sslocal/railtie.rb:9:in `block in <class:Railtie>'
```

Here's that railtie code:

```
initializer "sslocal.set_up_webpacker", :before => "webpacker.proxy" do
  state = SSLocal::State.new(Rails.env.to_s)
  Webpacker.config.dev_server[:https] = true if state.enabled?
end
```

I'm not very familiar with how railties work, so I'm not sure how the initializer is supposed to work if `webpacker.proxy` is not defined, but when I remove webpacker from my bundle, this block is still running, causing the error.

Changing the condition of the last line in the block to `if state.enabled? && defined?(Webpacker)` fixes this error for me though.

Seems like an easy fix, but if I should just move on to a different tool because this repo is sunsetting, no problem :)

**Comment by pat** [comment]
Hi @jaredmoody - thanks for reporting this! This gem is just kicking along, not abandoned, but not actively evolving. So: if you'd like to submit a PR with your change - or perhaps instead, add a line `next unless defined?(Webpacker)` above `state = ...` instead? - then I'll very happily merge and get a new release out.

**Comment by pat** [comment]
Just released 0.1.1 with your fix included :)

**Comment by jaredmoody** [comment]
Awesome, thanks!
